src/uct/ib/dc/dc_mlx5.c

```diff
@@ -440,6 +441,12 @@ ucs_status_t uct_dc_mlx5_iface_dci_connect(uct_dc_mlx5_iface_t *iface,
         return UCS_ERR_IO_ERROR;
     }
 
+    if (uct_ib_set_ece(dev, dci->txwq.super.verbs.qp,
+                       iface->super.super.config.ece) != UCS_OK) {
+        ucs_error("error setting qp ece");
```

Comment:
1. move this error print inside uct_ib_set_ece
2. use status variable

Reply:
Done